packages/embedder/ai_models/TextGenerationInterface.ts

```diff
@@ -0,0 +1,80 @@
+import {
```

Comment:
A thought for the future:

In the future, we could have deployments with local LLMs. It might be nice for us to have a generic GPT interface that abstracts away whether or not we are using a local LLM or OpenAI, etc. and have the executor services consume that interface rather than call OpenAI directly